libnopegl/meson.build

```diff
@@ -409,6 +409,8 @@ gbackends_cfg = {
     'src': files(
       'src/backends/gl/api_gl.c',
       'src/backends/gl/hwmap_gl.c',
+      'src/ngpu/opengl/cmd_buffer_gl.c',
+      'src/ngpu/opengl/fence_gl.c',
```

Comment:
Kinda unrelated but is the `_gl` suffix needed in a "opengl" subdirectory? I'm seeing a few inconsistencies btw, like "glcontext" and "glstate", "or program_gl_utils".

Not a blocking comment.

Reply:
I think it's better to keep the suffix as it matches the structure they contain + it avoids any confusion while dealing with #include directives, ie: "ngpu/ctx.h" vs "ngpu/opengl/ctx.h" vs "ctx.h" if you are already in the ngpu/opengl directory or the ngpu directory. 
Also, I took the decision to remove the prefix ngpu_ from the file but I'm not against bringing it back (some codestyle prefers to keep everything in sync, filename + structure + ifndef)

Reply:
For the glcontext / glstate inconsistency (note that we have the same for vulkan), it's actually quite a common naming / inconsistency in a lot of renderers. They are helpers, not implementations of a public NGPU structure, so I think it's fine that way.